### PR TITLE
Idyofficiall.wordpress.com

### PR DESCRIPTION
**Pull request by Ibrra77**
<script data-ad-client="ca-pub-3917347400664939" async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>

- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,4 +1,3 @@
----
-layout: index
-lang: en
----
```
Comment (mrymzf) [inline index.html:4]: Hack-whatsapp-messages-prank

Comment (mrymzf) [reply index.html:4]: https://github.com/github/opensource.guide/pull/2482#discussion_r675978566

Comment (mrymzf) [reply index.html:4]: 09101404305

**Comment by mrymzf** [comment]
09101404305:whatsapp

File: index.html (hunk 1 above)
Comment (mrymzf) [reply index.html:4]: Number to get chat

Comment (mrymzf) [reply index.html:4]: Alirezaebii148@gmail.com

**Review by Ibrra77 — COMMENTED** [review]
cc3f020e8fb2819044c8342ce11f7b87d1a0e082

**Pull request closed without merging**